packages/video_player/lib/video_player.dart

```diff
@@ -87,10 +135,18 @@ class VideoPlayerValue {
   /// Is null when [initialized] is false.
   final Size size;
 
+  /// False when [duration] is null, true otherwise.
```

Comment:
I feel like this is kind of explaining he implementation detail. If the implementation changes we will have to update the doc as well which is kinda of "high maintenance"?
Maybe something like `Indicates whether the video player is finished initialization and can be played`.